src/formatter.py

```diff
@@ -1,199 +1,98 @@
 """
-Bunyan Formatter.
+SagaFormatter
 
-Provides logging compatibility with Bunyan standard and enceforth CLI.
+Ensures logs are formatted as we want them.
+cfr: https://github.com/sagacify/logger
 """
 import datetime
-import json
-import logging
-import socket
-import traceback
 import os
+import socket
 import sys
-
-from inspect import istraceback
-
-# Support order in python 2.7 and 3
-try:
-    from collections import OrderedDict
-except ImportError:
-    pass
-
-
-def object_startswith(key, value):
-    return hasattr(key, 'startswith') and key.startswith(value)
+import time
+import json
 
 
-def merge_record_extra(record, target, reserved=[]):
-    """
-    Merges extra attributes from LogRecord object into target dictionary.
+class SagaEncoder(json.JSONEncoder):
+    def default(self, obj):
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) An attribute defined in json.encoder line 158 hides this method [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=Pylint%3AE0202)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This function has 5 returns or yields, which is more than the 3 allowed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1142)